topics.py

```diff
@@ -0,0 +1,63 @@
+def compare_subs(topic, mqtt_topic_prefix, target_device):
+    if topic == '{device_id}/shadow/get/accepted':
+        topic = target_device +'/shadow/get/accepted'
+    elif topic == '$aws/things/{device_id}/shadow/get/rejected':
+        topic = '$aws/things/' + target_device + '/shadow/get/rejected'
+    elif topic == '$aws/things/{device_id}/shadow/update/delta':
+        topic = '$aws/things/', target_device + '/shadow/update/delta'
+    elif topic == 'mqtt_topic_prefix/m/d/{device_id}/d2c':
+        topic = mqtt_topic_prefix + 'm/d/' + target_device + '/d2c'
+    elif topic == 'mqtt_topic_prefix/m/d/{device_id}/c2d':
+        topic = mqtt_topic_prefix + 'm/d/'+ target_device + '/c2d'
+    elif topic == 'matt_topic_prefix/{device_id}/jobs/rcv':
+        topic = mqtt_topic_prefix + target_device + '/jobs/rcv'
+    elif topic == 'm/#':
+        topic = mqtt_topic_prefix + 'm/#'
+    elif topic == 'a/connections':
+        topic = mqtt_topic_prefix + 'a/connections'
+    else:
+        topic = topic  #user manual input
+    print(topic)
+    return topic
```

Comment:
Consider using `{mqtt_topic_prefix}` for consistency with `{device_id}`

Also, one of these says `matt` rather than `mqtt`


If you adopt the above suggestion, then the function could be rewritten as:


```python
def compare_subs(topic, mqtt_topic_prefix, target_device):
	ret = topic.format(mqtt_topic_prefix = mqtt_topic_prefix, device_id=target_device)
	print(ret)
	return ret
```

Reply:
good catch on the `matt` issue, Georges

Reply:
Thank you for catching that!